WNPRC_EHR/resources/queries/study/_BreedingEncounterByBreedingId/.qview.xml

```diff
@@ -0,0 +1,15 @@
+<customView xmlns="http://labkey.org/data/xml/queryCustomView" label="Pregnancies - Current">
+    <columns>
+        <column name="date"/>
```

Comment:
Note that if you don't need to adjust the column list you can omit the <columns> element completely and just have the <sorts>